tests/plugin/test_override_by_external_plugins.py

```diff
@@ -0,0 +1,55 @@
+from textwrap import dedent
+
+import pytest
+
+
+def test_override_build_in_commands(runner, test_root_path, _install_plugin, caplog):
+    config_path = (
+        test_root_path / "test_data" / "configs" / "override_plugin_config.toml"
+    )
+
+    result = runner.invoke(["--config-file", config_path, "connection", "list"])
+
+    assert (
+        caplog.messages[0]
+        == "Cannot register plugin [override]: Cannot add command [snow connection list] because it already exists."
+    )
+    assert result.output == dedent(
+        """+---------------------------------------+
+| connection_name | parameters          |
+|-----------------+---------------------|
+| test            | {'account': 'test'} |
++---------------------------------------+
+    """
+    )
+
+
+def test_disabled_plugin_is_not_executed(
```

Comment:
as discussed let's add one more test case confirming that no code from disabled plugin's module is executed

we need to check that not only command implementation is not executed but we need to be sure that no code is executed